xcode/Subconscious/Shared/Models/MemoAddress.swift

```diff
@@ -105,8 +105,21 @@ extension MemoAddress {
         }
     }
     
-    func isProfile() -> Bool {
-        return self.slug.isProfile()
+    var isProfile: Bool {
+        self.slug.isProfile()
```

Comment:
Nit: should we eventually change `slug.isProfile` to a property?

No action needed.

Reply:
Changed it to a property, I think this is better.